pkg/types/authtypes/organization.go

```diff
@@ -0,0 +1,29 @@
+package authtypes
+
+import (
+	"slices"
+	"strings"
+
+	"github.com/SigNoz/signoz/pkg/errors"
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+)
+
+var _ Typeable = new(organization)
+
+type organization struct{}
+
+func (organization *organization) Tuples(subject string, relation Relation, selector Selector, parentTypeable Typeable, parentSelectors ...Selector) ([]*openfgav1.CheckRequestTupleKey, error) {
+	if !slices.Contains(typeOrganizationSupportedRelations, relation) {
+		return nil, errors.Newf(errors.TypeInvalidInput, ErrCodeAuthZUnsupportedRelation, "unsupported relation for type %s, supported relations are %v", TypeOrganization.StringValue(), typeOrganizationSupportedRelations)
+	}
```

Comment:
```suggestion
	if !slices.Contains(typeOrganizationSupportedRelations, relation) {
		return nil, errors.Newf(errors.TypeInvalidInput, ErrCodeAuthZUnsupportedRelation, "unsupported relation for type %s, supported relations are %v", TypeOrganization.StringValue(), typeOrganizationSupportedRelations)
	}
```

Is this necessary? Seems like something the role module should prohibit.